sharktank/pyproject.toml

```diff
@@ -2,6 +2,44 @@
 requires = ["setuptools", "wheel"]
 build-backend = "setuptools.build_meta"
 
+[project]
+name = "sharktank"
+authors = [
+  {name = "SHARK Authors"},
+]
+description = "SHARK layers and inference models for genai"
+readme = "README.md"
+license = {text = "Apache-2.0"}
+classifiers = [
+  "Development Status :: 3 - Alpha",
+  "License :: OSI Approved :: Apache Software License",
+  "Programming Language :: Python :: 3",
+  "Programming Language :: Python :: 3.11",
+  "Programming Language :: Python :: 3.12",
+  "Programming Language :: Python :: 3.13",
+]
+requires-python = ">= 3.11"
+dependencies = ["iree-turbine"]
+dynamic = ["version"]
+
+[project.optional-dependencies]
+testing = ["pytest"]
+
+[project.urls]
+Repository = "https://github.com/nod-ai/SHARK-Platform"
+
+[tool.setuptools.packages.find]
+where = ["."]
+include = ["sharktank*"]
+namespaces = true
+
+[tool.setuptools.package-data]
+sharktank = ["py.typed", "kernels/templates/*.mlir"]
+
+[tool.setuptools.dynamic]
+# version = {file = ["version_info"]}
+dependencies = {file = ["requirements.txt"]}
```

Comment:
Why is version commented out here? Could use a comment pointing to `setup.py` maybe?